pkg/commands/docker.go

```diff
@@ -136,6 +138,10 @@ func (t *dockerBuildTask) ScanForDependencies(runner build.Runner) ([]build.Imag
 func (t *dockerBuildTask) Build(runner build.Runner) error {
 	args := []string{"build"}
 
+	if t.hypervIsolation {
```

Comment:
add in an `isolation` parameter instead so we can specify it to whatever we want; e.g. process/namespace isolation is good enough for our use cases, no need for hyper-v partition isolation